### DISCUSSION TITLE
Config save count

### DISCUSSION THREAD

**Discussion by ghost** [tbnobody/OpenDTU#963] [Q&A]
Hi,
Switched from AHOY DTU to Open DTU, like it very much but..
( got the hardware from blinkyparts.com very nice)
How can I reset: ''Config save count'' in system info?

regards

**Comment by tbnobody** [comment]
Why do you like to reset it?  What's the benefit?
(You can do it by performing a factory reset, in this case your whole configuration will be deleted)

**Comment by ghost** [comment]
Thx
